docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -128,68 +52,31 @@ Remember to write a description.
 
     An example of a completed page to release the code
 
-3. Click "Publish release".
-
-Updating The sha256
---------------------
-
-1. Download "Source code (.tar.gz)" from the `GitHub releases page <https://github.com/E3SM-Project/e3sm_diags/releases>`_.
-
-2. Run ``shasum -a 256`` on this file. For example:
+5. If this version is a release candidate (``<version>`` appended with ``rc``), checkmark `This is a pre-release`.
+6. Click `Publish release`.
+7. CI/CD release workflow is automatically triggered.
 
-    ::
-
-        shasum -a 256 e3sm_diags-1.1.0.tar.gz
-
-3. On your machine, pull the latest version of the code.
-This will have the ``conda/meta.yaml`` we edited in the first section.
+Releasing On Anaconda
+---------------------
 
-    ::
-
-        git fetch <upstream-origin> master
-        git checkout -b <branch-name> <upstream-origin>/master
+#. Be sure to have already completed :ref:`Releasing on GitHub <github-release>`. This triggers the CI/CD release workflow that handles Anaconda releases.
```

Comment:
A neat trick with rst is using `#.` to automatically infer the numbers for ordered lists.

This is less tedious than manually updating bullet numbers when bullets are added or deleted.